airbyte-cdk/python/README.md

```diff
@@ -41,3 +41,6 @@ Find the reference docs for the Airbyte Specification (the interface for how sou
 **Simple Python connectors using the barebones `Source` abstraction**:
 * [Google Sheets](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-google-sheets/google_sheets_source/google_sheets_source.py)
 * [Mailchimp](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-mailchimp/source_mailchimp/source.py)
+
+### Contributing to the CDK
+* `MyPy` is installed locally. Perform static type checking by running `mypy airbyte_cdk`. `MyPy` configuration is in the `.mypy.ini` file.
```

Comment:
```suggestion
#### First time setup
Create a virtual env and install dependencies by running
`` `
python -m venv .venv
source .venv/bin/activate
pip install -r requirements.txt
`` `

#### Iteration
* Iterate on the code locally
* Run tests via `pytest -s unit_tests`
* Perform static type checks using using `mypy airbyte_cdk`. `MyPy` configuration is in `.mypy.ini`.
```

Reply:
MyPy is installed by the `airbyte-python` gradle task - which unfortunately isn't working right now: https://github.com/airbytehq/airbyte/issues/3192

Reply:
I'll add it to the `setup.py` so local Python devs don't need to think about gradle.

Reply:
yup totally agreed. A python dev should always be able to work in their native pythonic workflow